clang/include/clang/Driver/Options.td

```diff
@@ -4047,6 +4047,10 @@ def fsycl_libspirv_path_EQ : Joined<["-"], "fsycl-libspirv-path=">,
   Visibility<[ClangOption, CLOption, DXCOption, CC1Option]>, HelpText<"Path to libspirv library">;
 def fno_sycl_libspirv : Flag<["-"], "fno-sycl-libspirv">, Visibility<[ClangOption, CLOption, DXCOption]>,
   HelpText<"Disable check for libspirv">;
+def fsycl_use_spirv_backend_for_spirv_gen : Flag<["-"], "fsycl-use-spirv-backend-for-spirv-gen">,
```

Comment:
This is an experimental option. I am not sure if we need to get approval for this. Adding @mdtoguchi for comment. Thanks

Reply:
I think having the documentation here is fine.  Until we are ready to fully document and get the option syntax reviewed just mark with `[HelpHidden]`

Reply:
Thanks. I will add the marking.